html-forms exercise-Dylan De Vei

dylandevei opened the pull request:
https://user-images.githubusercontent.com/99229316/158461528-c1e2b0f6-afda-45f4-90fb-c2bd4542ce43.mov

dylandevei committed:
completed html-forms exercise.

codingcodymiller commented on html-forms/index.html:
```diff
@@ -0,0 +1,47 @@
+    </select>
+    <br>
+    <br>
+    <label for="language">Favorite Language (choose one)</label>
```
This element isn't actually labeling a single form control element, so it can just be a `p` element.

codingcodymiller commented on html-forms/index.html:
```diff
@@ -0,0 +1,47 @@
+    <label for="javascript">JavaScript</label>
+    <br>
+    <br>
+    <label for="information"> Would you like to sign up for more information?
```
This element isn't actually labeling a single form control element, so it can just be a `p` element.

dylandevei committed:
fixed the label elements for language and information and changed them the p elements.

dylandevei commented:
![Screen Shot 2022-03-15 at 1 22 13 PM](https://user-images.githubusercontent.com/99229316/158465312-5ed106b8-4689-4876-8601-a437d70d24b5.png)
Here is a screenshot of my updated pull request, I fixed the label elements for language and information and changed them to p elements, per requested. Thanks!

codingcodymiller commented on html-forms/index.html:
```diff
@@ -0,0 +1,40 @@
+    <input id="javascript" type="radio" name="language" value="javascript" />
+    <label for="javascript">JavaScript</label>
+    <p> Would you like to sign up for more information?</p>
+    <input id="information" type="checkbox" name="information" /> Sign me up!</label>
```
You seem to be missing the opening tag for this `label` element

dylandevei committed:
updated lines 31-32 and inserted the opening tag for the label element.

dylandevei commented:
![Screen Shot 2022-03-15 at 2 38 30 PM](https://user-images.githubusercontent.com/99229316/158477129-fc52a498-6244-44d8-aa4b-98b9f012a719.png)
Here is an updated screenshot from my most recent pull request. Here I added the missing opening tag for the label element. Thanks!

codingcodymiller commented on html-forms/index.html:
```diff
@@ -28,7 +28,8 @@ <h1>Coding Form!</h1>
     <p> Would you like to sign up for more information?</p>
-    <input id="information" type="checkbox" name="information" /> Sign me up!</label>
+    <input id="information" type="checkbox" name="information"/>
+    <label>Sign me up!</label>
```
This `label` needs a `for` attribute that matches the `id` attribute on its paired form control element.

dylandevei committed:
added for attribute to match id of paired form control element(information).

dylandevei commented:
![Screen Shot 2022-03-15 at 2 43 51 PM](https://user-images.githubusercontent.com/99229316/158478041-862a29f2-c0bc-439f-8e3b-2125e6ca2a6f.png)
